libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl

```diff
@@ -0,0 +1,31 @@
+domain ATQR
+
+context Challenges
+
+//função no Challenge
+
+💬 RequestJudgement? :challengeId
+  rules:
+    challenge.status = ChallengeStatus.Overdue
+
+  sendJudgementRequestEmail? :challengeId
```

Comment:
you cannot have a command inside another command

Reply:
como assim? não posso ter um comando dentro de outro comando? mas a gente faz isso o tempo inteiro, não? haha